src/nni_manager/training_service/kubeflow/kubeflowTrainingService.ts

```diff
@@ -570,15 +622,32 @@ class KubeflowTrainingService implements TrainingService {
         if(this.kubeflowTrialConfig) {
             switch(roleType) {
                 case 'ps':
-                    if(this.kubeflowTrialConfig.ps && this.kubeflowTrialConfig.ps.gpuNum == 0) {
-                        runScriptLines.push(`export CUDA_VISIBLE_DEVICES=''`);
+                    if(this.kubeflowTrialConfig instanceof KubeflowTrialConfigTensorflow){
```

Comment:
suggest to add a param for generateRunScript, like resetGPU : boolean. 
The only thing generateRuScript need to do is:
if(resetGPU) {
   runScriptLines.push(`export CUDA_VISIBLE_DEVICES=''`);
}